test/integration/decorators.ts

```diff
@@ -74,6 +74,18 @@ suite('TypeScript Decorators', function() {
       const propValue = testElement.readOnlyString;
       chai.assert.equal(propValue, 'initial value');
     });
+    
+    test('computed property should should return computed value', function() {
+      testElement.computedString = "new value";
```

Comment:
i guess this should be setting `reflectedString` to `yahoo`? so the following computed result is `computed yahoo`? by the looks of the test element source.

Reply:
yes